__tests__/app.test.js

```diff
@@ -438,3 +438,26 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET :article_id comment_count", () => {
+  test("should return comment_count where comment(s) match article_id ", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("comment_count");
+        expect(article.comment_count).toEqual("1");
```

Comment:
Really good only checking for the `comment_count` key here as that's all we're introducing! 2 things to consider:
- Is there a more appropriate matcher for primitives than toEqual?
- Do we typically expect a `count` to be a string? If you were to change this, you may find some other tests have an issue just to warn!